examples/instrumentation/main.go

```diff
@@ -17,22 +17,58 @@ limitations under the License.
 package main
 
 import (
+	"context"
+	"math/rand"
+	"time"
+
 	"github.com/caicloud/nirvana"
+	"github.com/caicloud/nirvana/definition"
 	"github.com/caicloud/nirvana/log"
 	"github.com/caicloud/nirvana/plugins/metrics"
 	"github.com/caicloud/nirvana/plugins/profiling"
 )
 
+// This example shows how metrics and profiling plugin work. And the defaults functionality they provide.
```

Comment:
nit: This example shows how metrics and profiling plugin work, and the defaults functionality they provide.